README.md

```diff
@@ -301,6 +301,8 @@ const handleIssuesOpenedEvent = (event: IssuesOpenedEvent) => {
 };
 ```
 
+**Note**: You _MUST_ enable the [`strictNullChecks` option](https://www.typescriptlang.org/tsconfig#strictNullChecks) in your `tsconfig` in order to use the types. See [#395](https://github.com/octokit/webhooks/issues/395) for details.
```

Comment:
I would try to rephrase this to sound less imperative but looks good to me.

```suggestion
**⚠️ Caution ⚠️ **: Webhooks Types are expected to be used with [`strictNullChecks` option](https://www.typescriptlang.org/tsconfig#strictNullChecks) in your `tsconfig`. If you don't have this option enabled, there's the possibility you get `never` as inferred type in some use cases. See [#395](https://github.com/octokit/webhooks/issues/395) for details.
```

Reply:
Updated to use your wording